README.md

```diff
@@ -0,0 +1,75 @@
+# Sharded Redis Client
+
+## Description
+
+Creates a dummy redis client that consistently shards by key using sha1
```

Comment:
I realize this is a silly comment, and the readme was copy-pasted, but I am not sure I agree with the use of "dummy" here, it's a real redis client.

Reply:
Yea, I agree, I'll take that out